workspace/viewer/PdfJsViewer.ts

```diff
@@ -515,49 +520,51 @@ export class PdfJsViewer {
 
   onResize() {
     if (this.currentFile) {
-      this.openCurrentPage();
+      this.showPage(this.currentPage);
     }
   }
 
   pageUp = () => {
-    if (this.currentPage > 1) {
-      this.currentPage -= 1;
-      this.openCurrentPage();
-    }
+      if (this.currentPage > 1) {
```

Comment:
Something is wrong with the indents. We use 2 spaces.

Reply:
Done.